kind/actions/kind.go

```diff
@@ -291,6 +292,9 @@ func writeKubeConfig(n *nodes.Node, dest string, hostAddress string, hostPort in
 }
 
 func image(version string) string {
+	if !strings.HasPrefix(version, "v") {
+		version = fmt.Sprintf("v%s", version)
```

Comment:
its weird we're hardcoding the prefix with a `v`.

Reply:
why is it weird?

Reply:
Missed the `!strings.HasPrefix(version, "v")` part. If we're doing that verification, can we also do some sorta semver parameter checking. 

Not a blocker.

Reply:
yeah, that's low on the priority list but would be good to make an issue for it